packages/ckeditor5-theme-lark/theme/theme.css

```diff
@@ -2,3 +2,83 @@
  * Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
+
+/* TODO: Order */
```

Comment:
The order of the imports is important here. For example, with the current order, the "Powered by CKEditor" banner isn't positioned correctly if there's content after the editor.

I need to spend some more time to find the optimal order, but maybe @oleq can point out any obvious problems with this order?